### PR TITLE
feat: guix, chimeraos and raspbian 12

### PR DESCRIPTION
**Pull request by cappsyco**
chimeraos not to be confused with chimera

### CODE REVIEW

File: chimeraos/1
Comment (SteveLauC) [inline chimeraos/1:1]: This file should be named "chimeraos" [considering that this distro is rolling-based](https://github.com/which-distro/os-release?tab=readme-ov-file#directory-and-file-naming-conventions)

File: guix/1 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+NAME="Guix System"
+ID=guix
```
Comment (SteveLauC) [inline guix/1:2]: According to the distrowatch, this distro is in a fixed-release model, but its os-release file does not contain any `VERSION` key😵‍💫

**Review by SteveLauC — COMMENTED** [review]
Hi, thanks for the PR, left some comments

File: guix/1 (hunk 1 above)
Comment (cappsyco) [reply guix/1:2]: there's a chance this is my fault as I believe I got this from a live install, so a fully installed version might have an ID included. I will double check and revert, but in the case they have no ID value is this file name correct to leave as is?

Comment (cappsyco) [reply guix/1:2]: Ok, the installed os-release file is just as barebones! I know from the website this is version 1.4.0 I can update the filename to that if it's preferable

Comment (SteveLauC) [reply guix/1:2]: Thanks for your time on checking a fully-installed distro! 

> I know from the website this is version 1.4.0 I can update the filename to that if it's preferable

Let's merge it as-is, I try to keep the file names consistent, but honestly, not every distro obeys the rule. 😪

**Review by SteveLauC — APPROVED** [review]
Thanks!

**Comment by cappsyco** [comment]
Sorry, just using this as a place to ask a quick question on your preferred pull request process.

I'm going to be going through a lot of distros that I need to add to my own fetch program so this will likely mean I'll have a lot of new files to add here. Do you prefer a pull request per new distro or will one big pull request with them all be suitable? Or something in the middle?

Basically, how can I be as minimally annoying as possible? :D

**Comment by SteveLauC** [comment]
> Sorry, just using this as a place to ask a quick question on your preferred pull request process.

No need to be sorry 😆

> Do you prefer a pull request per new distro or will one big pull request with them all be suitable? Or something in the middle?

The only preference I have is to keep the commits semantic, i.e., include the distros added in them, and to keep commit history linear, so if you do:

1. A PR per new distro, then it will be merged as one commit. You can contain many commits in this PR, but I will do squash merge to squash them to 1 commit, e.g., #59
2. A big PR containing them all, if so, it should be split into commits
    1. One commit for every distro ( this would be tiring😶)
    2. One commit containing all the distors, if the commit message can accommodate them, it is ok. If not, you can 
    3. Group them into commits so that a commit won't include too many distros that can not be accommodated in one commit message😵. 

For a PR that contains multiple commits, I would merge all the commits if possible because [contributions count by commits](https://github.com/which-distro/os-release/graphs/contributors), so this is probably something contributors care about if they want to make more contributions. If I am unable to do this, I will squash them into 1 commit.

> Basically, how can I be as minimally annoying as possible? :D

Thanks for asking, appreciate it. :)

**Comment by cappsyco** [comment]
Thanks! There's definitely some etiquette around commits I need to be more aware of so this is very helpful indeed.

**Pull request merged**